pandas/core/format.py

```diff
@@ -24,11 +24,11 @@
 import itertools
 import csv
 
-docstring_to_string = """
-     Parameters
-     ----------
-     frame : DataFrame
-         object to render
+common_docstring = """
+    Parameters
+    ----------
+    frame : DataFrame
```

Comment:
frame should come out, we don't normally document the type of the self paramater (and thus its confusing).